pages/home/script.js

```diff
@@ -36,15 +36,12 @@ let position =
     ? { lat: latitude, lng: longitude }
     : Map.DEFAULT_LOCATION;
 
-// let positionObj = {};
-// let locationDetails = '';
+let positionSecondary = {};
 let hazardCardParams = {};
 let searchSuggestions = [];
 let reports = [];
 let categoryFilters = [];
 let flyToTrigger = true;
-const FLY_TO_ZOOM = 12;
-const ANIMATION_DURATION = 4;
```

Comment:
these variables were not required as `flyTo` is a new function being used to fly to any location